app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -116,182 +119,145 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
   private var paymentDataSubject: ReplaySubject<AdyenCardWrapper>? = null
   private var paymentDetailsSubject: PublishSubject<AdyenComponentResponseModel>? = null
   private var adyen3DSErrorSubject: PublishSubject<String>? = null
-  private var isStored = false
-  private var askCVC = true
 
-  private val bindingCreditCardPreSelected: AdyenCreditCardPreSelectedBinding? by lazy {
-    if (isPreSelected) AdyenCreditCardPreSelectedBinding.bind(
-      requireView()
-    ) else null
-  }
   private val bindingCreditCardLayout: AdyenCreditCardLayoutBinding? by lazy {
-    if (!isPreSelected) AdyenCreditCardLayoutBinding.bind(
-      requireView()
-    ) else null
+    AdyenCreditCardLayoutBinding.bind(requireView())
```

Comment:
To keep the same pattern of the rest of the app, I would suggest change "bindingCreditCardLayout" just to "binding"